### PR TITLE
refactor: follow suggestions

### PR DESCRIPTION
**Pull request by MarioRodrigues10**

### CODE REVIEW

File: lib/atomic_web/live/home_live/components/follow_suggestions.ex [hunk 1]
```diff
@@ -1,12 +1,19 @@
 defmodule AtomicWeb.HomeLive.Components.FollowSuggestions do
   @moduledoc false
-  use AtomicWeb, :component
+  use AtomicWeb, :live_component
+
+  alias Atomic.Organizations
+
+  attr :current_user, :map,
+    required: true,
+    doc: "Name of the follow button."
```
Comment (joaodiaslobo) [inline lib/atomic_web/live/home_live/components/follow_suggestions.ex]: Change the docs

**Review by joaodiaslobo — CHANGES_REQUESTED** [review]

**Review by joaodiaslobo — CHANGES_REQUESTED** [review]
It seems like the organizations are being re-ordered after following one, it looks kinda odd. Can you take a look into that?

**Comment by MarioRodrigues10** [comment]
@joaodiaslobo , I had to refactor a lot of code, so double check everything.
I don't really like the name of the component that i used, so if you have suggestions to name it I have my ears open.

File: lib/atomic_web/live/home_live/components/organization.ex [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    required: true,
+    doc: "The organization to display."
+
+  attr :current_user, :map,
+    required: true,
+    doc: "The current user logged in."
+
+  attr :button_placeholder, :string,
+    default: "Follow",
+    doc: "The button placeholder."
```
Comment (joaodiaslobo) [inline lib/atomic_web/live/home_live/components/organization.ex]:
```suggestion
  attr :is_following, :bool,
    default: false,
    doc: "Current user follow state for the organization."
```

File: lib/atomic_web/live/home_live/components/organization.ex [hunk 3]
```diff
@@ -0,0 +1,106 @@
+defmodule AtomicWeb.HomeLive.Components.FollowSuggestions.Organization do
```
Comment (joaodiaslobo) [inline lib/atomic_web/live/home_live/components/organization.ex]:
```suggestion
defmodule AtomicWeb.HomeLive.Components.FollowSuggestions.Suggestion do
```

File: lib/atomic_web/live/home_live/components/follow_suggestions.ex [hunk 4]
```diff
@@ -1,12 +1,19 @@
 defmodule AtomicWeb.HomeLive.Components.FollowSuggestions do
   @moduledoc false
-  use AtomicWeb, :component
+  use AtomicWeb, :live_component
```
Comment (joaodiaslobo) [inline lib/atomic_web/live/home_live/components/follow_suggestions.ex]: Can this be a functional component?

**Review by joaodiaslobo — CHANGES_REQUESTED** [review]

**Review by joaodiaslobo — DISMISSED** [review]
LGTM! 🤘

**Review by joaodiaslobo — APPROVED** [review]
Still LGTM! 🤘

**Comment by MarioRodrigues10** [comment]
@joaodiaslobo  i had to rebase due to recent changes, can you approve again? Thanks :pray:

**Pull request merged**